src/java/org/apache/cassandra/utils/ObjectSizes.java

```diff
@@ -133,6 +134,16 @@ public static long sizeOnHeapExcludingData(ByteBuffer buffer)
         return BUFFER_EMPTY_SIZE;
     }
 
+    public static long sizeOfEmptyHeapByteBuffer()
+    {
+        return BUFFER_EMPTY_SIZE;
+    }
```

Comment:
Not directly related to your patch, but it seems a bit misleading that `sizeOnHeapExcludingData()` takes an argument. Perhaps it would be better to be explicit and just remove it, replacing current usages w/ your new `sizeOfEmptyHeapByteBuffer()` method.